docs/antora/modules/async-mode/pages/async_mode.adoc

```diff
@@ -1,45 +1,42 @@
-= Ring Buffer
+= Async Mode
```

Comment:
Doc from the docs/ root was moved into antora to replace a much older version. The original docs/async_mode now links to portal